src/CitizenAction/CitizenActionParticipantsExporter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AppBundle\CitizenAction;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CitizenActionParticipantsExporter
+{
+    public function export(array $participants): string
+    {
+        $handle = fopen('php://memory', 'r+');
```

Comment:
what about do this with a template twig
```twig
Col1;Col2;Col3;Col4
{% for row in data %}
    {{ row.field1 }};{{ row.field2 }};{{ row.field3 }};{{ row.field4 }}
{% endfor %}
``` 

```php
public function export(array $participants): string
{
  return $this->templating->renderView(
    'view_path.twig',
    ['data' => $participants]
  );
}
```


maybe directly in `*Controller::exportParticipantsAction`?

Reply:
It should be better to create a PR dedicated to this refacto because there is another class `Exporter` with the same logic.